spec/hybrid/async_spec.js

```diff
@@ -1,14 +1,10 @@
 describe('async angular2 application', function() {
```

Comment:
(Unrelated to this change but please fix): This describe should be changed to `async hybrid angular1/2 application`.